docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -554,6 +558,62 @@ The discard limit increases with level according to the table below.
 |   5   |      6      |   3   |
 |   8   |      8      |   4   |
 
+### Interacting with the Environment
+
+#### Light and Vision
+
+Certain types of Checks require light and vision to see the environment or a target
+effectively. The amount of light available impacts Checks for most creatures. There are some Powers that affect a creature's abilities depending on the amount of light available. 
+
+**Partial Darkness:** When making a Detection or Investigation Check in partial
+  darkness, draw with the Lower Hand.
+
+*Examples of Partial Darkness:*
+- A dark alley in a city with some light spilling over from a nearby street. 
+- An underground room with some candles burning lightly. 
+
+**Full Darkness:** When making a Check in full darkness, draw with the Lower Hand.
+
+*Examples of Full Darkness:* 
+- A moonless night in the countryside without any light source. 
+- Descending into a cave without a light source.
```

Comment:
Does this include knowledge checks? 
> Murder mystery. The lights go out. I believe the killer is in the room. I think I smell their perfume. Is that lavender? Draw knowledge check

What about Vitality checks? Do I bleed worse in the dark?

Maybe we change to same as above but lower hand 3
```suggestion
**Full Darkness:** When making any Check in full darkness, draw with the Lower Hand. For example ...
    - A moonless night in the countryside without any light source. 
    - Descending into a cave without a light source.
```

Reply:
I adjusted to be Lower Hand (3) on Detection or Investigation. With that adjustment Darkness does not affect Attacks, meaning in a pitch black room you can still aim perfectly at a nearby target (or one up to X feet away based on ranged attacks.) 

Might be worth us revisiting in playtesting if we encounter Darkness often and want to adjust the intensity of its effects, but resolving for now.

Reply:
No objection, but I'm thinking maybe attacks should also lower hand (2 or 3) in full darkness.